include/openthread/platform/radio.h

```diff
@@ -169,6 +180,21 @@ typedef struct otRadioIeInfo
     uint8_t mTimeSyncSeq;       ///< The Time sync sequence.
 } otRadioIeInfo;
 
+/**
+ * This structure represents an IEEE 802.15.4 short or extended Address.
+ *
+ */
+typedef struct otMacAddress
```

Comment:
Move this out of `radio.h` (since not used by any `otPlatRadio`)? 
Suggest adding this and `otMacAddressType` to utls/mac_frame.h`